client/swagger.gradle

```diff
@@ -0,0 +1,44 @@
+dependencies {
+    // Version controlled by dependency management plugin
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+    implementation 'org.glassfish.jersey.core:jersey-client'
+    implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
+    implementation 'org.glassfish.jersey.media:jersey-media-multipart'
+
+    implementation 'io.swagger.core.v3:swagger-annotations'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
+}
+
+def artifactGroup = "${group}.catalog"
+
+generateSwaggerCode {
+    inputFile = file('../common/openapi.yml')
+    language = 'java'
+    library = 'jersey2'
+
+    // For Swagger Codegen v3 on Java 16+
+    // See https://github.com/swagger-api/swagger-codegen/issues/10966
+    jvmArgs = ['--add-opens=java.base/java.util=ALL-UNNAMED']
+
+    components = [
+        apiDocs  : false, apiTests: false,
+        modelDocs: false, modelTests: false
+    ]
+
+    additionalProperties = [
+        modelPackage  : "${artifactGroup}.model",
+        apiPackage    : "${artifactGroup}.api",
+        invokerPackage: "${artifactGroup}.client",
+        dateLibrary   : 'java11',
```

Comment:
Is java11 an option now? These docs mention java8 covers jdk 1.8+: https://openapi-generator.tech/docs/generators/java/

Reply:
Oh it looks like it was added in August: https://newreleases.io/project/github/swagger-api/swagger-codegen/release/v3.0.28

Reply:
After looking at the openapi source (the docs are out of date) it looks like setting `dateLibrary` to `java11` also sets `java8` to `true`, so it looks like the java8 setting is redundant. I'll remove it.